src/api/records.tsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
 import axios, { AxiosError } from 'axios';
```

Comment:
Since React import is removed, is it safe to rename the file extension to `.ts`? Same for `records.test.tsx`.

Reply:
I might keep it as a `.tsx` file for a couple of reasons: 1. `channels.tsx` is a similar file and it makes sense for them to be named similarly and 2. it may be in the future we add more `select` functions that do generate JSX code and thus would require renaming back to `records.tsx`.

Reply:
I see. In that case I would be inclined to keep it as is as well.